pages/api/utils/api-utils.js

```diff
@@ -14,5 +14,27 @@
  * limitations under the License.
  */
 
+import * as nodeFetch from "node-fetch";
+
 export const getRodeUrl = () =>
   process.env.RODE_URL || "http://localhost:50051";
+
+const fetch = (endpoint, method, body) => {
+  const options = {
+    method,
+  };
+
+  if (body) {
+    options.body = typeof body === "object" ? JSON.stringify(body) : body;
```

Comment:
It doesn't seem to matter in practice, but it would be good to specify the content type here as well.